cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -534,3 +535,46 @@ For Kerberos-enabled Hadoop clusters:
     rm -rf /yarn/nm/usercache/cdap
 
   Restart CDAP after removing the usercache(s).
+
+.. _cloudera-configuration-highly-available:
+
+Enabling CDAP HA
+----------------
+In addition to having a :ref:`cluster architecture <admin-manual-install-deployment-architectures-ha>`
+that supports HA (high availability), these additional configuration steps need to followed and completed:
+
+CDAP Master
+...........
+- Install multiple instances of the CDAP Master on different nodes.
```

Comment:
Pertaining to my suggestion above, the CM-specific bullet points would be:
- Using the Cloudera Manager UI, add additional Role Instances of Role "CDAP Master Service" to additional machines
- Ensure each machine has all required Gateway roles.
- Start each CDAP Master Service role.